### PR TITLE
[ALLUXIO-2233] Update S3A min copy part size.

### PR DESCRIPTION
**Pull request by calvinjia**
https://alluxio.atlassian.net/browse/ALLUXIO-2233
Previously it defaulted to 100MB.

### CODE REVIEW

File: underfs/s3a/src/main/java/alluxio/underfs/s3a/S3AUnderFileSystem.java [hunk 1]
```diff
@@ -79,6 +79,9 @@
   /** Threshold to do multipart copy. */
   private static final long MULTIPART_COPY_THRESHOLD = 100 * Constants.MB;
 
+  /** Minimum size of each copy part. */
+  private static final long MULTIPART_COPY_SIZE = 5 * Constants.MB;
```
Comment (ifcharming) [inline underfs/s3a/src/main/java/alluxio/underfs/s3a/S3AUnderFileSystem.java]: Just curious: is this a recommended setting by AWS or just a magic number?

Comment (calvinjia) [inline underfs/s3a/src/main/java/alluxio/underfs/s3a/S3AUnderFileSystem.java]: This is the minimum part size and the same size we use for uploads. The default multipart copy threshold was originally 5GB (which reduced to 100MB), so it makes sense to reduce the minimum part size if we plan to use multipart copy for most files.

**Comment by aaudiber** [comment]
Does this have a negative impact on large files? Since we're changing the default we might want to defensively make it configurable

**Comment by calvinjia** [comment]
@aaudiber I didn't make the value configurable to keep in line with the other internal constants we have. Do you think we should support some kind of pass through configuration to under storages? I think it is an unrelated topic though

**Comment by aaudiber** [comment]
Keeping it a constant sgtm. I think pass through configuration could help with avoiding the explosion of configuration parameters, maybe we could do something like passing through parameters starting with "underfs.ufstype."

**Pull request merged**